### PR TITLE
Install Cassandra v1.2

### PR DESCRIPTION
**Pull request by sgerrand**
:information_desk_person: This change ensures the version of Cassandra to be installed matches the version stream of that running in the UAT and production environments.

Fixes FundingCircle/codas#323

### CODE REVIEW

File: bin/cassandra_version_check.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  if [[ $prompt == "y" || $prompt == "Y" ]]
+  then
+    brew uninstall cassandra
+    rm -rf /usr/local/var/lib/cassandra
+    rm -rf /usr/local/etc/cassandra
+  fi
+fi
+
+
+
```
Comment (sldblog) [inline bin/cassandra_version_check.sh]: :scissors:

Comment (sgerrand) [inline bin/cassandra_version_check.sh]: :bow:

File: bin/install [hunk 2]
```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
 
+set -e
```
Comment (sldblog) [inline bin/install]:
```
$ which nonexistent
nonexistent not found
$ echo $?
1
```
Will probably make the following line not work.

Comment (sgerrand) [inline bin/install]: :neckbeard: 
```bash
$ set -e
$ test -z $(which /this/does/not/exist) && echo 'ohai David'
ohai David
$ echo $?
0
```

Comment (sldblog) [inline bin/install]: Cool! :bow:  Thanks :)

File: bin/cassandra_version_check.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+set -e
+
+if brew list -1 | grep -q "^cassandra$"; then
+  echo "Warning: You are using the latest version of Cassandra, which is currently incompatible with CODAS."
+  read -p "Would you like to remove it and install version 1.2? [y/N] " prompt
+  if [[ $prompt == "y" || $prompt == "Y" ]]; then
+    brew uninstall cassandra
+    rm -rf /usr/local/var/lib/cassandra
+    rm -rf /usr/local/etc/cassandra
```
Comment (sldblog) [inline bin/cassandra_version_check.sh]: Should this also try and kill any running cassandra process?

**Comment by sldblog** [comment]
Until killing, this LGTM :+1:

**Comment by lostie** [comment]
:trophy:

**Pull request merged**